<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>824</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_7">
    <item>
     <widget class="QStackedWidget" name="mainStackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="preparationPage">
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QScrollArea" name="scrollKitsArea">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
              <horstretch>1</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="scrollAreaWidgetContents">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>258</width>
               <height>238</height>
              </rect>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QPushButton" name="mathButton">
                <property name="text">
                 <string>Математика</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="chemistryButton">
                <property name="text">
                 <string>Химия</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="astronomyButton">
                <property name="text">
                 <string>Астрономия</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="geographyButton">
                <property name="text">
                 <string>География</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="engeneeringButton">
                <property name="text">
                 <string>Инженерия</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="csButton">
                <property name="text">
                 <string>Информатика</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="otherKitButton">
                <property name="text">
                 <string>Прочие</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="kitsDescription" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
              <horstretch>2</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>Устанавливаемые пакеты</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>Описание</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeType">
                  <enum>QSizePolicy::Expanding</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_16" stretch="0,0">
                 <property name="sizeConstraint">
                  <enum>QLayout::SetDefaultConstraint</enum>
                 </property>
                 <property name="topMargin">
                  <number>0</number>
                 </property>
                 <item>
                  <widget class="QStackedWidget" name="packagesChoose">
                   <property name="currentIndex">
                    <number>0</number>
                   </property>
                   <widget class="QWidget" name="mathPage">
                    <layout class="QHBoxLayout" name="horizontalLayout_3">
                     <item>
                      <widget class="QScrollArea" name="mathScrollArea">
                       <property name="widgetResizable">
                        <bool>true</bool>
                       </property>
                       <widget class="QWidget" name="mathScrollAreaContents">
                        <property name="geometry">
                         <rect>
                          <x>0</x>
                          <y>0</y>
                          <width>216</width>
                          <height>141</height>
                         </rect>
                        </property>
                        <layout class="QVBoxLayout" name="verticalLayout_2"/>
                       </widget>
                      </widget>
                     </item>
                    </layout>
                   </widget>
                   <widget class="QWidget" name="chemistryPage">
                    <layout class="QVBoxLayout" name="verticalLayout_3">
                     <item>
                      <widget class="QScrollArea" name="chemistryScrollArea">
                       <property name="widgetResizable">
                        <bool>true</bool>
                       </property>
                       <widget class="QWidget" name="chemistryScrollAreaContents">
                        <property name="geometry">
                         <rect>
                          <x>0</x>
                          <y>0</y>
                          <width>768</width>
                          <height>96</height>
                         </rect>
                        </property>
                        <layout class="QVBoxLayout" name="verticalLayout_9"/>
                       </widget>
                      </widget>
                     </item>
                    </layout>
                   </widget>
                   <widget class="QWidget" name="astonomyPage">
                    <layout class="QHBoxLayout" name="horizontalLayout_6">
                     <item>
                      <widget class="QScrollArea" name="astronomyScrollArea">
                       <property name="widgetResizable">
                        <bool>true</bool>
                       </property>
                       <widget class="QWidget" name="astronomyScrollAreaContents">
                        <property name="geometry">
                         <rect>
                          <x>0</x>
                          <y>0</y>
                          <width>66</width>
                          <height>18</height>
                         </rect>
                        </property>
                        <layout class="QVBoxLayout" name="verticalLayout_8"/>
                       </widget>
                      </widget>
                     </item>
                    </layout>
                   </widget>
                   <widget class="QWidget" name="geographyPage">
                    <layout class="QVBoxLayout" name="verticalLayout_10">
                     <item>
                      <widget class="QScrollArea" name="geographyScrollArea">
                       <property name="widgetResizable">
                        <bool>true</bool>
                       </property>
                       <widget class="QWidget" name="geographyScrollAreaContents">
                        <property name="geometry">
                         <rect>
                          <x>0</x>
                          <y>0</y>
                          <width>218</width>
                          <height>172</height>
                         </rect>
                        </property>
                        <layout class="QVBoxLayout" name="verticalLayout_11"/>
                       </widget>
                      </widget>
                     </item>
                    </layout>
                   </widget>
                   <widget class="QWidget" name="engeneeringPage">
                    <layout class="QHBoxLayout" name="horizontalLayout_5">
                     <item>
                      <widget class="QScrollArea" name="engeneeringScrollArea">
                       <property name="widgetResizable">
                        <bool>true</bool>
                       </property>
                       <widget class="QWidget" name="engeneeringScrollAreaContents">
                        <property name="geometry">
                         <rect>
                          <x>0</x>
                          <y>0</y>
                          <width>218</width>
                          <height>172</height>
                         </rect>
                        </property>
                        <layout class="QVBoxLayout" name="verticalLayout_12"/>
                       </widget>
                      </widget>
                     </item>
                    </layout>
                   </widget>
                   <widget class="QWidget" name="csPage">
                    <layout class="QVBoxLayout" name="verticalLayout_13">
                     <item>
                      <widget class="QScrollArea" name="csScrollArea">
                       <property name="widgetResizable">
                        <bool>true</bool>
                       </property>
                       <widget class="QWidget" name="csScrollAreaContents">
                        <property name="geometry">
                         <rect>
                          <x>0</x>
                          <y>0</y>
                          <width>66</width>
                          <height>18</height>
                         </rect>
                        </property>
                        <layout class="QVBoxLayout" name="verticalLayout_14"/>
                       </widget>
                      </widget>
                     </item>
                    </layout>
                   </widget>
                   <widget class="QWidget" name="otherKitsPage">
                    <layout class="QHBoxLayout" name="horizontalLayout_7">
                     <item>
                      <widget class="QScrollArea" name="otherKitsScrollArea">
                       <property name="widgetResizable">
                        <bool>true</bool>
                       </property>
                       <widget class="QWidget" name="otherKitsScrollAreaContents">
                        <property name="geometry">
                         <rect>
                          <x>0</x>
                          <y>0</y>
                          <width>66</width>
                          <height>18</height>
                         </rect>
                        </property>
                        <layout class="QVBoxLayout" name="verticalLayout_15"/>
                       </widget>
                      </widget>
                     </item>
                    </layout>
                   </widget>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="kitName">
                   <property name="font">
                    <font>
                     <pointsize>12</pointsize>
                    </font>
                   </property>
                   <property name="text">
                    <string>Математика</string>
                   </property>
                   <property name="scaledContents">
                    <bool>true</bool>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignCenter</set>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QTextEdit" name="packageDescription">
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                 <property name="html">
                  <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Open Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="downloadButton">
          <property name="toolTip">
           <string/>
          </property>
          <property name="statusTip">
           <string/>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Начать загрузку</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="downloadPage">
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QTextEdit" name="installationLog">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="returnButton">
          <property name="text">
           <string>Вернуться назад</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>824</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
